internal/cephfs/core/volume.go

```diff
@@ -268,39 +265,22 @@ func (s *subVolumeClient) ExpandVolume(ctx context.Context, bytesQuota int64) er
 	return err
 }
 
-// ResizeVolume will try to use ceph fs subvolume resize command to resize the
-// subvolume. If the command is not available as a fallback it will use
-// CreateVolume to resize the subvolume.
+// ResizeVolume will use ceph fs subvolume resize command to resize the subvolume.
 func (s *subVolumeClient) ResizeVolume(ctx context.Context, bytesQuota int64) error {
 	newLocalClusterState(s.clusterID)
```

Comment:
`newLocalClusterState(s.clusterID)` also need to be removed

Reply:
Keeping this for the time being.